extensions/attention-stream/public/options.js

```diff
@@ -6,50 +6,74 @@ const PAUSED = "Paused";
 const DB_NAME = "attention-stream";
 
 function changeState(state) {
-    if (state === RUNNING) {
-        document.getElementById("status").textContent = "Running";
-        document.getElementById("toggleEnabled").checked = true;
-    } else if (state === PAUSED || state === undefined) {
-        document.getElementById("status").textContent = "Paused";
-        document.getElementById("toggleEnabled").checked = false;
-    } else {
-        console.error("Unknown state:", state);
-    }
+  if (state === RUNNING) {
+    document.getElementById("status").textContent = "Running";
+    document.getElementById("toggleEnabled").checked = true;
+  } else if (state === PAUSED || state === undefined) {
+    document.getElementById("status").textContent = "Paused";
+    document.getElementById("toggleEnabled").checked = false;
+  } else {
+    console.error("Unknown state:", state);
+  }
 }
 
 // Update UI to current state.
-browser.storage.local.get("state").then(storage => changeState(storage.state));
+browser.storage.local
+  .get("state")
+  .then((storage) => changeState(storage.state));
 
 // Listen for state changes.
 browser.storage.onChanged.addListener((changes) => {
-    if (changes.state && changes.state.newValue) {
-        changeState(changes.state.newValue);
-    }
+  if (changes.state && changes.state.newValue) {
+    changeState(changes.state.newValue);
+  }
 });
 
-document.getElementById("toggleEnabled").addEventListener("click", async event => {
+document
+  .getElementById("toggleEnabled")
+  .addEventListener("click", async (event) => {
     if (event.target.checked === true) {
-        browser.runtime.sendMessage({ type: "rally-sdk.change-state", data: { state: "resume" } });
+      browser.runtime.sendMessage({
+        type: "rally-sdk.change-state",
+        data: { state: "resume" },
+      });
     } else {
-        browser.runtime.sendMessage({ type: "rally-sdk.change-state", data: { state: "pause" } });
+      browser.runtime.sendMessage({
+        type: "rally-sdk.change-state",
+        data: { state: "pause" },
+      });
     }
-});
+  });
 
 document.getElementById("download").addEventListener("click", async () => {
-    const db = new Dexie(DB_NAME);
-    await db.open();
+  const db = new Dexie(DB_NAME);
+  await db.open();
 
-    for (const tableName of ["user-journey", "advertisements", "article-contents"]) {
-        const tableContents = await db.table(tableName).toArray();
-        const dataUrl = (`data:application/json,${encodeURIComponent(JSON.stringify(tableContents, null, 2))}`);
+  for (const tableName of [
+    "user-journey",
+    "advertisements",
+    "article-contents",
+    "youtube-video-details",
+    "youtube-video-recommendations",
+    "youtube-ads",
```

Comment:
This approach is getting a bit unwieldy, maybe we should file a followup to put these all together in a single .zip to download or something.

Reply:
ooh yea that's a good idea! every time I click download it's like "ahhhHGGH!!"

just filed: https://github.com/mozilla-rally/rally/issues/225